.travis.yml

```diff
@@ -11,20 +13,21 @@ branches:
 language: generic
 
 env:
-  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=10.15.3
-  - NPM_TAG=next IMAGE_NAME=theia-full NODE_VERSION=10.15.3
-  - NPM_TAG=latest IMAGE_NAME=theia NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-java NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-java NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-go NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-go NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=yangster NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=yangster NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-python NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-python NODE_VERSION=10
-  - NPM_TAG=latest IMAGE_NAME=theia-swift NODE_VERSION=10
-  - NPM_TAG=next IMAGE_NAME=theia-swift NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=10.15.3
+#  - NPM_TAG=next IMAGE_NAME=theia-full NODE_VERSION=10.15.3
+#  - NPM_TAG=latest IMAGE_NAME=theia NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=theia NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-java NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=theia-java NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-go NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=theia-go NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=yangster NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=yangster NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-python NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=theia-python NODE_VERSION=10
+#  - NPM_TAG=latest IMAGE_NAME=theia-swift NODE_VERSION=10
+#  - NPM_TAG=next IMAGE_NAME=theia-swift NODE_VERSION=10
+   - NPM_TAG=next IMAGE_NAME=theia-rust NODE_VERSION=Dummy
```

Comment:
Is there a reason for having `NODE_VERSION=Dummy`?

Reply:
My image is directly based on Ubuntu and an installation of yarn using apt-get. This automatically installs node 8.  I will look into "nvm" to see if I can install the node version + yarn requested by NODE_VERSION

Reply:
I updated it to use nvm. It nos respects NODE_VERSION